### DISCUSSION TITLE
Home Assistant OS 11.3

### DISCUSSION THREAD

**Discussion by agners** [home-assistant/operating-system#3038] [Announcements]
## Home Assistant Operating System

* Enable IP policy routing options (#3026) (#3027) @agners
* Bump buildroot to update dbus-broker to 35 (#3008) @agners
* Enable WPA3 support in wpa_supplicant to support WPA3-SAE (#3000) @agners

## Build System

* Suppress action-shellcheck warning from deprecated option (#3030) @sairon
* Generate self-signed certificate in the prepare step and archive it (#3015) @sairon
* Add script for checking resulting kernel config (#3006) @sairon
* Adjust build to allow running from forks or feature branches (#2998) @sairon
* Add build summary for each board build (#2990) @sairon
* Add Raspberry Pi 5 to GitHub Action (#2985) @agners
* Add image size check to hdd-image.sh (#2965) @sairon
* Prepare 11.3.rc1 (#3013) @agners
* Bump actions/upload-artifact from 3 to 4 (#2997) @dependabot
* Bump actions/download-artifact from 3 to 4 (#2996) @dependabot
* Bump actions/setup-python from 4 to 5 (#2981) @dependabot
* Bump actions/stale from 8.0.0 to 9.0.0 (#2980) @dependabot

## Raspberry Pi

* Set Raspberry Pi 2 Supervisor architecture correctly (#3012) @agners
* Add rtw89, mt7921 and mt7922 WiFi and BT firmware (#3003) @sairon
* Enable ZSTD compression of firmware files (#3001) @sairon
* Remove Raspberry Pi 1/Zero from docs (#2986) @sairon
* Add Raspberry Pi 5 to GitHub Action (#2985) @agners
* Initial Raspberry Pi 5 support (#2914) @agners
* Bump brcmfmac_sdio-firmware-rpi to sync with RPi OS (#2971) @sairon

## Home Assistant Yellow

* Add rtw89, mt7921 and mt7922 WiFi and BT firmware (#3003) @sairon
* Enable ZSTD compression of firmware files (#3001) @sairon
* Bump brcmfmac_sdio-firmware-rpi to sync with RPi OS (#2971) @sairon

## Home Assistant Green

* Linux: Update kernel 6.1.70 (#3029) @sairon
* Add rtw89, mt7921 and mt7922 WiFi and BT firmware (#3003) @sairon
* Enable ZSTD compression of firmware files (#3001) @sairon

## Open Virtual Appliance

* Linux: Update kernel 6.1.70 (#3029) @sairon
* Add rtw89, mt7921 and mt7922 WiFi and BT firmware (#3003) @sairon
* Enable virtio-mem and DMI tables/fw_cfg sysfs for aarch64 and ova (#3004) @sairon
* Enable ZSTD compression of firmware files (#3001) @sairon

## Generic x86-64

* Linux: Update kernel 6.1.70 (#3029) @sairon
* Enable driver for Realtek card readers (#3005) @sairon
* Add rtw89, mt7921 and mt7922 WiFi and BT firmware (#3003) @sairon
* Enable ZSTD compression of firmware files (#3001) @sairon

## Hardkernel ODROID

* Linux: Update kernel 6.1.70 (#3029) @sairon
* Add rtw89, mt7921 and mt7922 WiFi and BT firmware (#3003) @sairon
* Enable ZSTD compression of firmware files (#3001) @sairon

## ASUS Tinker

* Linux: Update kernel 6.1.70 (#3029) @sairon
* Add rtw89, mt7921 and mt7922 WiFi and BT firmware (#3003) @sairon
* Enable ZSTD compression of firmware files (#3001) @sairon

## Khadas VIM Series

* Linux: Update kernel 6.1.70 (#3029) @sairon
* Add rtw89, mt7921 and mt7922 WiFi and BT firmware (#3003) @sairon
* Enable ZSTD compression of firmware files (#3001) @sairon

## Generic aarch64

* Linux: Update kernel 6.1.70 (#3029) @sairon
* Add rtw89, mt7921 and mt7922 WiFi and BT firmware (#3003) @sairon
* Enable virtio-mem and DMI tables/fw_cfg sysfs for aarch64 and ova (#3004) @sairon
* Enable ZSTD compression of firmware files (#3001) @sairon

<hr /><em>This discussion was created from the release <a href='https://github.com/home-assistant/operating-system/releases/tag/11.3'>Home Assistant OS 11.3</a>.</em>

**Comment by PCServices2** [comment]
I am running 11.3rc1 on a Pi5. 
How do I get the 11.3 stable version?
I have tried the usual check for updates in the uni and tried “ha os update” and “ha os upgrade” in the terminal (I have no idea is either of these commands are valid). 

Thanks in advance.

**Reply by sairon** [reply]
Because of how the deployment process works, 11.3 was not available for CLI or UI update at the time of publishing this discussion post - it needs to be manually approved first after we make sure the build is healthy. It was added to the version repository just 4 hours ago: https://github.com/home-assistant/version/commit/af3c0ed802663352e559bd75fa1107e9dd668136. You should be able to update to that version now.

**Comment by leonardpitzu** [comment]
@agners the 11.3 pi5 release contains the 12.0 supervisor which cannot restore backups done on 12.1 which is the latest release. 
edit: wrong, 12.1 is in beta thus you have to manually update to it before restoring from backup... mea culpa

**Reply by PCServices2** [reply]
Thanks.

I have just found that the update has now become available through the fronted Update service.

**Comment by corvy** [comment]
Will this release be included in the Raspberry Pi Imager for Raspberry Pi 5? It still is not selectable I see.

**Reply by sairon** [reply]
We still consider RPi5 build as a "developer preview". Once we're sure it's suitable for general public, we'll add it also to the Raspberry Pi Imager.

**Reply by corvy** [reply]
Ok. I will "Balena" in the meantime 🔥

**Reply by danuw** [reply]
I was sure I saw it in the pi imager earlier this week. I checked again today and it was not. Was it added by mistake and then removed please?

**Reply by sairon** [reply]
@danuw It was never there. The source of truth for available images is here, you can check yourself: https://github.com/home-assistant/version/commits/master/rpi-imager-haos.json

**Comment by cj-github-dev** [comment]
I imaged a SSD connected to my RPI5 via USB to HAOS 11.3 using https://github.com/home-assistant/operating-system/releases/download/11.3/haos_rpi5-64-11.3.img.xz

It starts booting fine, but then reaches a point where i get:

[FAILED] Failed to start Docker Application Container Engine.
See 'systemctl status docker.service' for details.

It flies through that so fast and immediately thereafter stops all services, unmounts everything and reboots, so I'm not sure how to look into that status.

**Comment by folta** [comment]
Upgrading to HAOS 11.3 on my HA Yellow broke the bluetooth built into my Pi CM4. The device address was changed to "00:00:00:00:00:00" and would fail to initialize. Rolling back to 11.2 restored the functionality.

```
Could not determine the power state of the Bluetooth adapter hci0 [00:00:00:00:00:00] due to timeout after 5 seconds
```

```
Socket read <event_code=CommandStatusEvent, controller_idx=0, param_len=3> <command_opcode=ReadControllerInformation, status=InvalidIndex>
```

My assumption is a link to the CM4 bluetooth changes that disables CM4 bluetooth when no built in bluetooth is supposed to be present (base CM4 module).
